state/postgresql/postgresdbaccess.go

```diff
@@ -131,12 +131,13 @@ func (p *postgresDBAccess) setValue(req *state.SetRequest) error {
 			ON CONFLICT (key) DO UPDATE SET value = $2, isbinary = $3, updatedate = NOW();`,
 			tableName), req.Key, value, isBinary)
 	} else {
-		// Convert req.ETag to integer for postgres compatibility
-		var etag int
-		etag, err = strconv.Atoi(*req.ETag)
+		// Convert req.ETag to uint32 for postgres XID compatibility
+		var etag64 uint64
```

Comment:
You reference uint32 in the comment, but that's a uint64. Does it matter?

Reply:
`uint64` is used because that is what is returned by `strconv.ParseUint` so the `64` suffix was to make that explicit.